server/src/constants.ts

```diff
@@ -6,11 +6,12 @@ let platformDir =
 // See https://microsoft.github.io/language-server-protocol/specification Abstract Message
 // version is fixed to 2.0
 export let jsonrpcVersion = "2.0";
+export let platformPath = path.join("rescript", platformDir);
```

Comment:
TODO: check that `platformDir` computation is in sync with the compiler.

Reply:
Not quite, in rescript-vscode the `platformDir` also allows for "linuxarm64", the compiler does not:
https://github.com/rescript-lang/rescript-compiler/blob/master/bsc#L6-L12
https://github.com/rescript-lang/rescript-compiler/blob/master/scripts/bin_path.js#L8-L11

Reply:
In that case the compiler is a little behind and could be updated.

Reply:
Or is the logic in rescript-vscode correct? The compiler has no directories for `win32arm64` or `linuxarm64`, and neither do the get built in CI?

Reply:
It would be kind of future-proof, if one wants to build from source.
But does not matter much either way, as this is rarely done I guess.